rxandroid/src/main/java/rx/android/lifecycle/LifecycleObservable.java

```diff
@@ -72,6 +76,50 @@ public Boolean call(LifecycleEvent lifecycleEvent) {
     }
 
     /**
+     * Similar to {@link #bindActivityLifecycle(Observable, Observable)} but uses
+     * {@link android.app.Application.ActivityLifecycleCallbacks} provided in
+     * API 14 to listen for Activity lifecycle.
+     *
+     *</p>
+     * This helps figuring out the corresponding next lifecycle event in which to unsubscribe.
+     * <pre>
+     * {@code
+     *  class MyActivity extends Activity {
+     *    protected void onStart() {
+     *      super.onStart();
+     *      subscription = LifecycleObservable.bindActivityLifecycle(
+     *          this,
+     *          ViewObservable.clicks(button),
+     *          LifecycleEvent.START)
+     *         .subscribe(...);
+     *   }
+     *  }
+     * }
+     * </pre>
+     *
+     * @param activity the activity we want to monitor lifecycle sequence for
+     * @param source   the source sequence
+     * @param bindEvent the binding {@link LifecycleEvent} associated with the <code>source</code> Observable.
```

Comment:
Why do I pass in the lifecycle event in which I'm doing the binding instead of passing in the lifecycle event which should terminate the subscription? The latter seems much more intuitive and declarative of intent.